Core/GDCore/Extensions/Builtin/CameraExtension.cpp

```diff
@@ -590,6 +590,19 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsCameraExtension(
       .SetDefaultValue("\"\"")
       .AddParameter("expression", _("New default Z order"));
 
+  extension
+      .AddExpressionAndCondition("number",
+                                 "LayerHighestZOrder",
+                                 _("Highest Z order of a layer"),
+                                 _("the highest Z order of a layer"),
+                                 _("the highest Z order of objects in the layer _PARAM1_"),
```

Comment:
```suggestion
                                 _("the highest Z order of objects in the layer"),
                                 _("the highest Z order of objects in the layer _PARAM1_"),
```

Reply:
I prefer keeping `a layer` in this sentence, because of the phrasing here:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/32449369/193814940-152bb2b6-4777-4a54-adce-92a3f3f47b50.png">
But I'll add `objects in`